### PR TITLE
[master] Fix complaints from the broken link checker

### PR DESCRIPTION
**Pull request by LukeShu**
Some of these are actual broken links, but some of them are just removing
the leading "https://www.getambassador.io/" from links, as that makes it
hard to distinguish between the deployed website and website previews.

### CODE REVIEW

File: about/why-ambassador.md [hunk 1]
```diff
@@ -46,14 +46,14 @@ Ambassador takes full advantage of Kubernetes and Envoy Proxy.
 
 ## For more information
 
-[Deploy Ambassador today](https://www.getambassador.io/user-guide/install) and join the community [Slack Channel](http://d6e.co/slack).
+[Deploy Ambassador today](../../user-guide/install) and join the community [Slack Channel](http://d6e.co/slack).
 
-If you're interested in commercial support and additional features, see [Ambassador Pro](https://www.getambassador.io/pro).
+If you're interested in commercial support and additional features, see [Ambassador Pro](/pro).
```
Comment (scoyle391) [inline about/why-ambassador.md]: should this go to an aes link instead?

File: concepts/auth-overview.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ The implementation of an effective authentication strategy is vital to any appli
 
 Typically with web applications the authentication is implemented at the edge, either via an API/edge gateway, or via a top-level request filter within your application framework. It is also increasingly common for applications to use external identity providers -- such as Google, GitHub, or Facebook -- typically via an Identity hub like [Auth0](https://auth0.com/), [Keycloak](https://www.keycloak.org/) or [Okta](https://www.okta.com/) that provides authentication-as-a-service, rather than taking on the high cost (and risk) of maintaining their own identity database.
 
-This article is focused on implementing authentication at the edge with the Kubernetes-native [Ambassador Pro API gateway](https://www.getambassador.io/pro/), and shows an example of how to integrate this with third-party identity providers.
+This article is focused on implementing authentication at the edge with the Kubernetes-native [Ambassador Pro API gateway](/pro/), and shows an example of how to integrate this with third-party identity providers.
```
Comment (scoyle391) [inline concepts/auth-overview.md]: same - we should figure out what to do with Pro links.

File: about/why-ambassador.md (hunk 1 above)
Comment (LukeShu) [reply about/why-ambassador.md]: No, because this is for the current GA version of the docs

File: concepts/auth-overview.md (hunk 2 above)
Comment (LukeShu) [reply concepts/auth-overview.md]: The Pro links have already been removed in the EA version of the docs, right?

Comment (scoyle391) [inline concepts/auth-overview.md]: you have this as (/pro/) but in some other places its (/pro)

Comment (LukeShu) [reply concepts/auth-overview.md]: Trailing slashes don't matter.

File: user-guide/tls-termination.md [hunk 3]
```diff
@@ -14,7 +14,7 @@ This guide requires you have the following installed:
 
 ## Install Ambassador
 
-Install Ambassador in Kubernetes using either [YAML manifests](https://www.getambassador.io/user-guide/getting-started) or the [Helm chart](https://www.getambassador.io/user-guide/helm). If you do not know which to do, use the YAML manifests.
+Install Ambassador in Kubernetes using either [YAML manifests](../../user-guide/getting-started) or the [Helm chart](../../user-guide/helm). If you do not know which to do, use the YAML manifests.
```
Comment (scoyle391) [inline user-guide/tls-termination.md]: so since tls-termination is in user-guide already, shouldn't these be ../ and not ../../?

Comment (LukeShu) [reply user-guide/tls-termination.md]: I could have simplified it to `../getting-started` from `../../user-guide/getting-started` (and likewise for `helm`), but it doesn't really matter.

**Comment by scoyle391** [comment]
Okay lgtm now!

**Pull request merged**